### ISSUE TITLE
Fix "inside-block-and-after-rule" example for rule-empty-line-before

### ISSUE THREAD

**Issue by AlexanderKositsyn** [stylelint/stylelint#4723]
**Clearly describe the documentation issue**

The description of "inside-block-and-after-rule" secondary option in "rule-empty-line-before" describes not correct behavior. [Docs](https://stylelint.io/user-guide/rules/rule-empty-line-before#inside-block-and-after-rule) says that black lines at  "a" class and "b" class will be removed. But as I can get form the description 
> "Reverse the primary option for rules that are inside a block and follow another rule."

that only class "b" suits for this statement. Because "b" follows another rule and inside a block media.
So i have tested it with following settings:
```json
"rule-empty-line-before": [
            "always-multi-line",
            {"except": ["inside-block-and-after-rule"]}
        ]
```
and i have got this error:
![image](https://user-images.githubusercontent.com/28633471/80320769-0ce73a00-8821-11ea-848c-98b66ba113e9.png)
As you can see "baz" class hasn't error because it meets two conditions (inside a block and follow another rule), meanwhile "boo" doesn't meet "follow another rule", so it has error.

**What solution would you like to see?**
I suggest to rewrite the "not considered violations" as 
```less
@media {

  a {}
  b {}
}
```

**Comment by jeddy3** [comment]
@AlexanderKositsyn Thanks for the report and for using the template.

It's a good catch. Will be fixed by #4725.
